lib/avromatic/model/attributes.rb

```diff
@@ -84,7 +84,12 @@ def add_validation(field)
 
         def add_required_validation(field)
           if required?(field) && field.default == :no_default
-            validates(field.name, presence: true)
+            case field.type.type_sym
+            when :array, :map
+              validates(field.name, exclusion: { in: [nil], message: "can't be nil" })
```

Comment:
Looks like the behavior was already correct here, but I switched booleans to the same nil check.